lib/tests/test_fix.rs

```diff
@@ -326,6 +326,50 @@ fn test_unchanged_file_is_fixed() {
     assert_eq!(*new_commit_b.tree_id(), expected_tree_b.id());
 }
 
+/// If a descendant is already correctly formatted, it should by rebased without
```

Comment:
typo: "be rebased". I'd say "its snapshot should be preserved and not inherit any fixes done to the parent".

Reply:
Done (slightly rephrased)